### DISCUSSION TITLE
Swap A & B Memory card option and shortcut key for boot from drive

### DISCUSSION THREAD

**Discussion by disksystem** [emukidid/swiss-gc#517] [Ideas]
Hi. I like playing the game with 480p and widescreen from the real disk drive.
But this method is a little bit complicated.

Like the situation, Let's play "Mario MARIO KART: Double Dash" with friends.

1. Insert exploit memory card to the slot A.
2. Put the "The Legend of Zelda: The Wind Waker" disc to the drive. 
3. Repeat pressing Start button.
4. Replace the memory card with real play data to slot A.
5. Replace the "Mario MARIO KART: Double Dash" disc to the drive.
6. Move the swiss menu to the disc drive.
7. Boot the game.

If there is a function "Swap A & B Memory card option and shortcut key for boot from drive"

1. Insert exploit memory card to slot A, and real memory card to slot B.
2. Put the "The Legend of Zelda: The Wind Waker" disc to the drive.
3. Repeat pressing Start button.
4. Put the "Mario MARIO KART: Double Dash" disc to the drive.
5. Boot by press shortcut button.

**Comment by vaderag** [comment]
You can use standard saves on the same card as exploit saves, so couldn't you just save MK:DD on the same card as for the Zelda hack
NB. I've never used the Zelda hack as I use Shinehax but do share the mem card
